app/helpers/ineligible_reasons_helper.rb

```diff
@@ -0,0 +1,40 @@
+module IneligibleReasonsHelper
+  def ineligible_reasons(cfe_result)
+    return "" unless cfe_result.assessment_result == "ineligible"
+
+    reasons = ineligible_reasons_array(cfe_result)
+    return reasons.first if reasons.length == 1
+
+    ineligible_reasons = ":<ul class='govuk-list govuk-list--bullet'>"
+    reasons.each { |reason| ineligible_reasons += "<li>#{reason}</li>" }
+    ineligible_reasons += "</ul>"
+  end
+
+private
+
+  def ineligible_reasons_array(cfe_result)
+    ineligible_reasons_array = []
+    ineligible_reasons_array << "gross income" if ineligible_gross_income?(cfe_result)
+    ineligible_reasons_array << "disposable income" if ineligible_disposable_income?(cfe_result)
+    ineligible_reasons_array << "disposable capital" if ineligible_disposable_capital?(cfe_result)
+    ineligible_reasons_array
+  end
```

Comment:
**not a blocker**: as mentioned below, even this method could belong to the backend/model and use keys rather than displayable values (such as `gross_income`). The presentation layer can then "translate" those keys. This maintains a "separation of concerns" IMO.

Reply:
Have moved the ineligible reasons into translations